readme.md

```diff
@@ -524,6 +524,53 @@ test.only.serial(...);
 
 This means you can temporarily add `.skip` or `.only` at the end of a test or hook definition without having to make any other changes.
 
+### Test macros
+
+Additional arguments passed to the test declaration will be passed to the test implementation. This is useful for creating reusable test macros.
+
+```js
+function macro(t, input, expected) {
```

Comment:
@jamestalmage Good thing we decided not to pass the context as the second argument 😝